projects/ng-components/lib/tooltip/tooltip.component.ts

```diff
@@ -29,10 +30,19 @@ export class TooltipComponent implements AfterViewInit {
     /** CSS class */
     @Input() className?: string = "";
 
+    /** Close the tooltip once the user scrolls */
+    @Input() closeOnScroll: boolean = false;
+
+    /** Delay before closing on scroll */
+    @Input() closeOnScrollDelay: number = 0;
+
+    /** reposition tooltip with more placement choices */
```

Comment:
Still, not clear enough, I don't understand what this does

Reply:
added more description